pkg/audit/manager.go

```diff
@@ -245,16 +245,16 @@ func (am *Manager) audit(ctx context.Context) error {
 		var res []Result
 		am.log.Info("Auditing from cache")
 		res, errs := am.auditFromCache(ctx)
+		if errs != nil {
+			am.log.Error(err, "Auditing")
```

Comment:
why do we want to log these errs twice, here and then few lines later?

Reply:
I missed that! fixed it